src/database/Database.ts

```diff
@@ -36,6 +36,32 @@ class Database {
     return this._client.db(`${this._dbName}-${guildId}`);
   }
 
+  async getDocValue(
+    guildId: string,
+    collectionId: string,
+    docId: string,
+  ): Promise<unknown> {
+    const doc = await this.getDb(guildId)
+      .collection(collectionId)
+      .findOne({ _id: docId });
+    return (doc || {}).value;
+  }
+
+  async setDocValue(
+    guildId: string,
+    collectionId: string,
+    docId: string,
+    value: any,
+  ): Promise<void> {
+    await this.getDb(guildId)
+      .collection(collectionId)
+      .updateOne(
+        { _id: docId },
+        { $set: { _id: docId, value } },
+        { upsert: true },
```

Comment:
These parameters actually alter the command so that the insertion applies only when there is no existing document. If there is an existing document, no update is performed. Since the method is named `setDocValue`, we could just do a normal `updateOne` like so:
```suggestion
        { _id: docId },
        value,
        { upsert: true },
```